test/system/add_patient_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'application_system_test_case'
+
+class AddPatientTest < ApplicationSystemTestCase
```

Comment:
can we rename this test to reflect what it's doing a little better? maybe `EventLogInteractionTest` (and a renamed file to match)? `AddPatient` sounds like it's doing a patient create to me. Thoughts?